config/initializers/0_ldap_authenticatable.rb

```diff
@@ -0,0 +1,109 @@
+require 'net/ldap'
```

Comment:
the fact that this is required to run before initializers/devise feels like a code smell and a bad hack.